components-core/src/main/java/com/adyen/checkout/components/core/paymentmethod/ACHDirectDebitPaymentMethod.kt

```diff
@@ -18,6 +18,7 @@ import org.json.JSONObject
 @Parcelize
 data class ACHDirectDebitPaymentMethod(
     override var type: String?,
+    @Deprecated("This property is deprecated. Use the new sdkData property instead.")
```

Comment:
`sdkData` is not really accessible as it is encoded, so I don't think merchants can use it. It seems like we don't have an alternative to offer and it's probably not necessary to offer that. Should we change the message to something like below?
`This property is deprecated and will be removed in future versions`